cores/rp2040/SerialUSB.cpp

```diff
@@ -174,8 +175,19 @@ SerialUSB::operator bool() {
 static bool _dtr = false;
 static bool _rts = false;
 static int _bps = 115200;
+static volatile bool _rebooting = false;
```

Comment:
Pretty sure `rebooting` doesn't need to be volatile here.  Only a USB callback ever sets it, and those are guaranteed to not be re-entrant.